erdjs/src/smartcontracts/typesystem/namespace.ts

```diff
@@ -0,0 +1,20 @@
+import { FunctionDefinition } from "./function";
+import { StructureDefinition } from "./structure";
+
+export class Namespace {
```

Comment:
Can you add a short description on what `Namespace` is for?

Reply:
Added.